### PR TITLE
add internal_route_vip_range to cc route syncer job

### PR DESCRIPTION
**Pull request by jeffpak**
[#161849257]

Thanks for contributing to the `capi_release`. To speed up the process of reviewing your pull request please provide us with:

* A short explanation of the proposed change:
These changes include a PR to cloud controller that allow the VIPs it generates to be sent to envoy. In addition to that PR, we are also introducing some bosh links to the cc route syncer job from cloud controller. The cloud controller job already provides the field `internal_route_vip_range`, which the route syncer needs in order to correctly determine VIPs. 

* An explanation of the use cases your change solves
This change allows the cc route syncer to successfully accept VIPs from cloud controller. This change also makes it so that we no longer use hard-coded vips, which prevents collision.

* Links to any other associated PRs
https://github.com/cloudfoundry/cloud_controller_ng/pull/1361

* [x] I have viewed signed and have submitted the Contributor License Agreement

* [x] I have made this pull request to the `develop` branch

* [x] I have run CF Acceptance Tests on bosh lite

Regards,
The CF Networking Team

### CODE REVIEW

File: jobs/cc_route_syncer/templates/cloud_controller_ng.yml.erb [hunk 1]
```diff
@@ -62,6 +62,18 @@ database_encryption:
   pbkdf2_hmac_iterations: <%= link("cloud_controller_internal").p("cc.database_encryption.experimental_pbkdf2_hmac_iterations") %>
 <% end %>
 
+<% if_link("cloud_controller_container_networking_info") do |info|
+  internal_vip_range = "#{info.p("cc.internal_route_vip_range")}"
+  raise StandardError.new("invalid cc.internal_route_vip_range: #{internal_vip_range}") unless internal_vip_range =~ /\A (?:\d{1,3}\.){3} \d{1,3} \/ \d{1,3} \z/x
```
Comment (tcdowney) [inline jobs/cc_route_syncer/templates/cloud_controller_ng.yml.erb]: Can you add some template tests for this behavior?

https://github.com/cloudfoundry/capi-release/blob/develop/spec/cloud_controller_ng/cloud_controller_ng_spec.rb

**Comment by tcdowney** [comment]
Any updates @jeffpak, let us know if you have any questions about the template test. Or ask @tylerschultz the Chief Architect® of our BOSH™ template testing.

File: jobs/cc_route_syncer/templates/cloud_controller_ng.yml.erb (hunk 1 above)
Comment (jeffpak) [reply jobs/cc_route_syncer/templates/cloud_controller_ng.yml.erb]: Added!

**Comment by tcdowney** [comment]
Changes look good to us, thanks @jeffpak!

**Pull request merged**